src/Type/MixedType.php

```diff
@@ -685,6 +685,14 @@ public function tryRemove(Type $typeToRemove): ?Type
 		return null;
 	}
 
+	public function exponentiate(Type $exponent): Type
+	{
+		return new BenevolentUnionType([
```

Comment:
What if all numeric types are subtracted?

Reply:
pow() supports a lot of types as exponent, see e.g. https://3v4l.org/ZtZVc

we would need a big union, which might not be really useful?